arm/Microsoft.RecoveryServices/vaults/protectionContainers/readme.md

```diff
@@ -1,18 +1,19 @@
-# RecoveryServicesProtectionContainer `[Microsoft.RecoveryServices/vaults/protectionContainers]`
+# RecoveryServices Vaults ProtectionContainers `[Microsoft.RecoveryServices/vaults/protectionContainers]`
```

Comment:
```suggestion
# Recovery Services Vaults Protection Containers `[Microsoft.RecoveryServices/vaults/protectionContainers]`
```

Reply:
Done

Reply:
Not sure, I can see the correct code in my branch

Reply:
See other comment